src/main/java/de/unijena/cheminf/clustering/desccalc/Descriptor.java

```diff
@@ -1822,6 +1832,40 @@ public static int getNumberOfComponents(
         }
```

Comment:
comment on getSpecifiedDescriptors(): I suggest removing the isQuicklyCalculableDescriptorInclusion parameter to make the overall concept cleaner. The other parameters are ex-/including descriptors with unfavourable properties (unsafe, slow, fingerprints). I don't think there is a use case where you only want the slow descriptors.